wcf.targets

```diff
@@ -47,4 +47,9 @@
      <WcfToolsOutputPath>$(WcfRootFolder)bin\Wcf\tools\</WcfToolsOutputPath>
   </PropertyGroup>
 
+  <!-- Override ValidateIgnoreReference from Target 'ValidatePackageTargetFramework' in PackageLibs.targets -->
+  <ItemGroup>
+    <ValidateIgnoreReference Include="System.Private.ServiceModel" />
+  </ItemGroup>
+
```

Comment:
Also, ValidateIgnoreReference is only something we want to use for classic assemblies.  It's essentially saying that these assemblies don't follow the rules of generations so don't expect to be able to calculate their generation.  For S.P.SM it should be a normal assembly built on contracts, so you should be able to validate its generation.